modules/api/pkg/handler/v2/external_cluster/external_cluster.go

```diff
@@ -103,12 +105,20 @@ func (req createClusterReq) Validate() error {
 }
 
 func DecodeManifestFromKubeOneReq(encodedManifest string) (*kubeonev1beta2.KubeOneCluster, error) {
+	var err error
+	var manifest []byte
+	rxBase64 := regexp.MustCompile(Base64)
```

Comment:
This could be a global var for performance, as `regexp.MustCompile(Base64)` needs to run only once.